sdk/communication/communication-chat/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@azure/abort-controller": "^2.1.2",
     "@azure/communication-common": "^2.3.1",
-    "@azure/communication-signaling": "1.0.0-beta.29",
+    "@azure/communication-signaling": "1.0.0-beta.33",
```

Comment:
Is this an GA version signaling release?

Reply:
This is beta version release.